x-pack/plugins/ml/public/jobs/new_job/simple/components/enable_model_plot_checkbox/enable_model_plot_checkbox_view.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+import PropTypes from 'prop-types';
+import React, { Fragment, Component } from 'react';
+
+import {
+  EuiCallOut,
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIconTip,
+} from '@elastic/eui';
+
+
+export class EnableModelPlotCheckbox extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      checked: false,
+    };
+  }
+
+  tooltipContent = `Select to enable model plot. Stores model information along with results.
+                  Can add considerable overhead to the performance of the system.`;
+
+  warningTitle = 'Proceed with caution!';
+
+  onChange = e => {
```

Comment:
nit, when an arrow function uses curlies, we tend to wrap the parameters in parentheses.
`onChange = (e) => {`
https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#arrow-functions